.travis.yml

```diff
@@ -1,22 +1,22 @@
 dist: trusty
 language: node_js
 node_js:
-  - '8'
+  - '11'
 env:
   - MOZ_HEADLESS=1
 addons:
   chrome: stable
   firefox: latest
 if: (tag AND tag =~ ^v) OR (commit_message =~ ^Merge\spull\srequest\s#) OR (type = pull_request)
 before_install:
-  - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.6.0
+  - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.21.1
   - export PATH="$HOME/.yarn/bin:$PATH"
   - .travis/before_install.sh
   - "google-chrome-stable --headless --disable-gpu --remote-debugging-port=9222 http://localhost &"
 install: yarn --offline
 script:
-  - lerna run lint
-  - lerna run test
+  - yarn run lint
+  - yarn run test
```

Comment:
would yarn workspaces pick that up and run in all packages?

Reply:
That runs the scripts from the root package.json, which in turn run `lerna run x` ... so yeah ... this just makes the environment more consistent